### PR TITLE
Sync 'Feature/sphinx documentation' with main

### PR DESCRIPTION
**Pull request by frankcorneliusmartin**
Could you read the `community planning` section in DevOps and see if you agree?

### CODE REVIEW

**Review by bartvanb — APPROVED** [review]
One minor comment, but otherwise looks good

File: Makefile [hunk 1]
```diff
@@ -18,7 +18,8 @@ help:
 	@echo "  publish-test : publish built python packages to test.pypi.org"
 	@echo "  publish      : publish built python packages to pypi.org (BE CAREFUL!)"
 	@echo "  clean        : clean all built packages"
-	@echo "  community    : Notify community FLAGS="--version 99.99.88 --notes 'I should have done more!' --post-notes 'Oh.. Maybe not'""
+	@echo "  community    : notify community FLAGS="--version 99.99.88 --notes 'I should have done more!' --post-notes 'Oh.. Maybe not'""
+	@echo "  make         : run all unittests and compute coverage"
```
[the remark this answers is not in the document]
Comment (bartvanb) [reply Makefile:22]: And actually, why is this on this branch anyway?

[the remark this answers is not in the document]
Comment (frankcorneliusmartin) [reply Makefile:22]: Lol. Idk. I think the help is wrong in this case. Ill verify it. Thanks!

[the remark this answers is not in the document]
Comment (frankcorneliusmartin) [reply Makefile]: changed help to  `make test`

**Review by bartvanb — APPROVED** [review]
nice

**Pull request merged**